src/azure-cli/azure/cli/command_modules/appservice/_validators.py

```diff
@@ -128,6 +128,14 @@ def validate_app_exists(cmd, namespace):
     if not app:
         raise ResourceNotFoundError("'{}' app not found in ResourceGroup '{}'".format(app, resource_group_name))
 
+def validate_functionapp_on_containerapp_vnet(cmd, namespace):
```

Comment:
I'm starting to think this validation might have to be in the backend so that other client tools also have an error message for the customer.

Reply:
it's a good point! I think we can bring this up in the centauri channel.